### ISSUE TITLE
Heap Sort in Typescript

### ISSUE THREAD

**Issue by Danushka96** [FOSS-UCSC/FOSSALGO#630]
This is an open issue for the Heap Sort algorithm implementation in Typescript. If you wish to contribute, just make some noise and we will assign you.

**Comment by Jeewantha95** [comment]
Can i contribute?

**Comment by aayushbest** [comment]
I would to contribute to this issue.

**Comment by RavinduSachintha** [comment]
Due to @Jeewantha95 is already assigned to another issue, I assigned @aayushbest for this issue. Please read the contribution doc before starting the implementation. 👍
